### PR TITLE
SettingsView

### PR DESCRIPTION
**Pull request by slavny11**
Add settingsView and provisional views (ie MailCompose for modal and WebView for showing webpage inside app). Add function in SettingsView in order to manage all connections (settings, external website, mail).
MailComposer doesn't work yet. Needed to turn on one in developer account and signing and capabilities. Now it crash. I commented this function so far it's not super important.
AppStore warning shows up, but we don't have an app in appstore, so link after this doesn;t work.
We need to add infor about Vit D and new icon for Terms of use (we didn;t have this in figma).

Apart all limitations that I mentioned above I think we need to merge now into develop and I will fix this issues later. 

The main change that can affect another branches is NavigationalView in @main. I need one that we can make native transitions and NavLinks.

### CODE REVIEW

**Comment by slavny11** [comment]
I merged develop into this branch - the issue is only with NavigationalView inside @main. We need to fix it and later I'll manage all the other minor problems from the new branch.

File: RayD/RayD/Common/App/RayDApp.swift [hunk 1]
```diff
@@ -15,10 +15,12 @@ struct RayDApp: App {
 
     var body: some Scene {
         WindowGroup {
-            if isTheFirstAppBoot {
-                OnboardingView(mainViewModel: mainViewModel, isOnBoardingActive: $isTheFirstAppBoot)
-            } else {
-                ContentView(viewModel: mainViewModel)
+            NavigationView {
```
Comment (palant-dev) [inline RayD/RayD/Common/App/RayDApp.swift:18]: Have you tried putting navigation view only to the stack in Main View? Why we need it in the onboarding. If it works with no issue I'm fine with this.

File: RayD/RayD/UI/MainView/Subviews/SettingsView.swift [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+                        Image(systemName: "chevron.forward")
+                            .foregroundColor(.black)
+                            .opacity(0.3)
+                    }
+                }
+            }
+            Section {
+                Button {
+                    //                    sendEmail()
+#warning("Need to add mail in capabilities. Now it crashes app")
```
Comment (palant-dev) [inline RayD/RayD/UI/MainView/Subviews/SettingsView.swift:98]: May you add a .disabled to the button with decresed opacity just for not letting the user tap on it for now cause we don't know how much time we have in the next days

File: RayD/RayD/UI/MainView/Subviews/SettingsView.swift [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+                            .scaledToFit()
+                            .frame(width: 24, height: 24, alignment: .center)
+                        Text("About Vitamin D")
+                            .font(.system(size: 18, weight: .semibold))
+                            .foregroundColor(.black)
+                            .opacity(0.8)
+                    }
+                }
+
+                Button {
```
Comment (palant-dev) [inline RayD/RayD/UI/MainView/Subviews/SettingsView.swift:131]: Same here

**Review by palant-dev — CHANGES_REQUESTED** [review]
See comments

**Review by yukim17 — APPROVED** [review]

File: RayD/RayD/UI/MainView/Subviews/SettingsView.swift (hunk 2 above)
Comment (slavny11) [reply RayD/RayD/UI/MainView/Subviews/SettingsView.swift]: Agree. Added

**Review by palant-dev — APPROVED** [review]

File: RayD/RayD/Common/App/RayDApp.swift (hunk 1 above)
Comment (slavny11) [reply RayD/RayD/Common/App/RayDApp.swift:18]: hello

**Review by palant-dev — APPROVED** [review]

**Pull request merged**